static/js/popovers.js

```diff
@@ -500,6 +493,21 @@ exports.register_click_handlers = function () {
         e.preventDefault();
     });
 
+    var clip = new Clipboard('.copy_link');
+    clip.on('error', function () {
+        blueslip.debug("Copying link failed");
+    });
```

Comment:
Linter was throwing an error due to `new`, so I had to define `var clip` and use it in `clip.on(error,....`. Any other way to tackle it.